bots/approve.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+
+import os
+import logging
+import yaml
+
+from github import Github
+from github import UnknownObjectException
+
+from thoth.common import init_logging
+
+
+__module__ = 'thoth.bots.approve'
+__version__ = '0.2.0-dev'
+
+DEBUG = bool(os.getenv('DEBUG', False))
+TEST = bool(os.getenv('TEST', False))
+SESHETA_GITHUB_ACCESS_TOKEN = os.getenv('SESHETA_GITHUB_ACCESS_TOKEN', None)
+CICD_CONTEXT_ID = 'continuous-integration/jenkins/pr-merge'
+
+if DEBUG:
+    logging.basicConfig(level=logging.DEBUG)
+else:
+    logging.basicConfig(level=logging.INFO)
+
+init_logging()
```

Comment:
The `init_logging()` call will overwrite the logging configuration above. Something like this should fix it (note that the order is significant):

```python
init_logging()
_LOGGER.getLogger(__module__)

if DEBUG:
    _LOGGER.setLevel(logging.DEBUG)
else:
     _LOGGER.setLevel(logging.INFO)
```